src/components/auth/BasicAuth/BasicAuthFlow.tsx

```diff
@@ -36,6 +36,15 @@ export function BasicAuthFlow({
             username: user,
             password: pass,
           },
+          ...(providerMetadata && {
+            providerMetadata: Object.entries(providerMetadata).reduce((acc, [name, value]) => {
```

Comment:
This technically works, but this would be a lot clearer as as a map function rather than a reduce function.